flags.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+)
+
+type kwargStringFlag struct {
+	value   *map[string][]byte
+	changed bool
+}
+
+func newKwargStringFlag() *kwargStringFlag {
+	return &kwargStringFlag{
+		value: new(map[string][]byte),
+	}
+}
+
+func (f *kwargStringFlag) AsMap() map[string][]byte {
+	var m map[string][]byte
+	if f.value != nil {
+		return *f.value
+	}
+	return m
```

Comment:
why not just return nil and delete L20?

Reply:
Good point. There used to be more code here so now it's unneeded.